README.md

```diff
@@ -74,9 +74,9 @@ vars:
 ```
 
 ### Disabling Models
-Your connector may not be syncing all tabes that this package references. This might be because you are excluding those tables. If you are not using those tables, you can disable the corresponding functionality in the package by specifying the variable in your `dbt_project.yml`. The metrics from the disabled tables will not populate in downstream models. By default, all packages are assumed to be true. You only have to add variables for tables you want to disable, like so:
+It is possible that your Salesforce connector does not sync every table that this package expects. If your syncs exclude certain tables, it is because you either don't use that functionality in Salesforce or actively excluded some tables from your syncs. 
 
-The `salesforce__user_role_enabled` variable below refers to the `user_role` table. 
+To disable the corresponding functionality in this package, you must add the corresponding variable(s) to your `dbt_project.yml`, which are listed below. By default, that is if none of these variables are added, all variables are assumed to be true. Add variables only for the tables you would like to disable:
 
 ```yml
 # dbt_project.yml
```

Comment:
You don't have to include lines 82-86 with the commented out dbt_project.yml and config-version: 2 line.

Reply:
Thanks, and I have updated. I also updated similarly in a few other similar code blocks for consistency.